src/main/java/storage/StorageHelper.java

```diff
@@ -0,0 +1,13 @@
+package storage;
+
+import author.AuthorList;
+
+//@@author xenthm
+/**
+ * This utility class simplifies calls to save the <code>authorList</code> to the data file.
+ */
+public class StorageHelper {
+    public static void saveFile(AuthorList authorList) {
+        Storage.getInstance().saveAuthorListToDataFile(authorList);
+    }
```

Comment:
The `saveFile` function will be used throughout the program, thus it is important for the call to be clear. Excellent job of abstracting this function, despite seeming trivial!